### PR TITLE
Download tests in only one process.

### PR DESCRIPTION
**Pull request by gordonmessmer**

### CODE REVIEW

File: tests/test_scripts_pythondistdeps.py [hunk 1]
```diff
@@ -196,7 +197,12 @@ def check_and_install_test_data():
                                               f"Possible resolution: Specify the package version with"
                                               f" trailing zeros in test-requires.yaml", file=sys.stderr)
 
-                                shutil.move(info_path, install_path)
+                                try:
+                                    # str() needed for Python < 3.9
+                                    shutil.move(str(info_path), str(install_path))
+                                except shutil.Error:
+                                    # when tests run in parallel, this happens
+                                    pass
```
Comment (hroncok) [inline tests/test_scripts_pythondistdeps.py]: In that case, the trying is no longer required.

File: tests/test_scripts_pythondistdeps.py [hunk 2]
```diff
@@ -216,7 +222,9 @@ def check_and_install_test_data():
 @pytest.fixture(scope="session", autouse=True)
 def fixture_check_and_install_test_data():
     """Wrapper fixture, because a fixture can't be called as a function."""
-    check_and_install_test_data()
+    with open(TEST_DATA_PATH.joinpath('test-requires.yaml.lock'), 'w') as f:
```
Comment (hroncok) [inline tests/test_scripts_pythondistdeps.py:221]: Suggestion: Gitignore that file?

Comment (gordonmessmer) [reply tests/test_scripts_pythondistdeps.py:221]: Both good points.  Updated.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 __pycache__/
 tests/data/scripts_pythondistdeps/usr/
+test-requires.yaml.lock
```
Comment (hroncok) [inline .gitignore]:
```suggestion
tests/data/scripts_pythondistdeps/test-requires.yaml.lock
```

**Review by hroncok — APPROVED** [review]

**Comment by hroncok** [comment]
Thanks a lot!

**Pull request merged**